src/main/java/build/buildfarm/worker/Executor.java

```diff
@@ -190,15 +210,14 @@ private Code executeCommand(
     }
 
     long startNanoTime = System.nanoTime();
-    int exitValue = -1;
     Process process;
     try {
       process = processBuilder.start();
       process.getOutputStream().close();
     } catch(IOException ex) {
       ex.printStackTrace();
       // again, should we do something else here??
-      resultBuilder.setExitCode(exitValue);
+      resultBuilder.setExitCode(exitCode);
```

Comment:
Sorry, I'm having a code-read fail -- can the exitCode here ever be not -1?

Reply:
It cannot, the exitCode assignments after the default initialization all occur after this. lmk if you think that merits different behavior here

Reply:
Oh, then I would very much prefer you just say -1 here, to not make the reader think about what the value might be.

Reply:
Pedantically, I used the exitCode here to match the releaseExecutor information propagated to the ExecuteActionStage so that there would not be any discontinuity between what is [logged](https://github.com/bazelbuild/bazel-buildfarm/pull/191/files/5009aa01810a4f392e314db27561f5d4af2efa10#diff-830c4529fc2f6fa022b5fad4505c0507R164) and what is recorded in the resultBuilder, but I see the point.